agentdesk/device_v1.py

```diff
@@ -50,7 +52,8 @@ class StorageStrategy(Enum):
 
 class ConnectConfig(BaseModel):
     agentd_url: Optional[str] = None
-    instance: Optional[str] = None
+    instance: Optional[str] = None #instance can be either a json stringified V1DesktopInstance or a searchable instance name
```

Comment:
why not just `V1DesktopInstance | str` ?

Reply:
yea I thought about it, and it is easier so went with it!